cmd/operator-sdk/cleanup/packagemanifests.go

```diff
@@ -0,0 +1,62 @@
+// Copyright 2019 The Operator-SDK Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cleanup
+
+import (
+	"fmt"
+	"path/filepath"
+
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+
+	olmcatalog "github.com/operator-framework/operator-sdk/internal/generate/olm-catalog"
+	olmoperator "github.com/operator-framework/operator-sdk/internal/olm/operator"
+	"github.com/operator-framework/operator-sdk/internal/util/projutil"
+)
+
+type packagemanifestsCmdLegacy struct {
+	olmoperator.PackageManifestsCmd
+}
+
+func newPackageManifestsCmdLegacy() *cobra.Command {
```

Comment:
add a comment to mark it as deprecated, to describe when and where it used as well,